src/main/java/it/gov/pagopa/pu/sil/registry/extrainfo/RegistryExtraInfoHandlerPaaSILImportaDovuto.java

```diff
@@ -0,0 +1,39 @@
+package it.gov.pagopa.pu.sil.registry.extrainfo;
+
+import it.gov.pagopa.pu.sil.registry.RegistryLogger;
+import it.veneto.regione.pagamenti.ente.ElementoListaDovutiEntiSecondari;
+import it.veneto.regione.pagamenti.ente.PaaSILImportaDovuto;
+import it.veneto.regione.pagamenti.ente.PaaSILImportaDovutoRisposta;
+import org.springframework.stereotype.Service;
+import org.springframework.ws.soap.SoapHeaderElement;
+
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+
+@Service
+public class RegistryExtraInfoHandlerPaaSILImportaDovuto {
+  public Map<String, Object> extractRequestExtraInfo(PaaSILImportaDovuto request, SoapHeaderElement header) {
+    Map<String, Object> body = new HashMap<>();
+    body.put("flagGeneraIuv", request.isFlagGeneraIuv());
+    if (request.getDovuto() != null && request.getDovuto().length > 0) {
+      body.put("dovuto", new String(request.getDovuto(), StandardCharsets.UTF_8));
```

Comment:
are we sure that it will produce a readable string? or it should be Base64 encoded?